Apps/W1/HybridGP/app/src/Migration/Vendors/GPVendorMigrator.codeunit.al

```diff
@@ -525,6 +537,115 @@ codeunit 4022 "GP Vendor Migrator"
         RemitAddress.Modify();
     end;
 
+    local procedure CreateRecurringPurchaseLines(var GPVendor: Record "GP Vendor"; var Sender: Codeunit "Vendor Data Migration Facade")
+    var
+        GPCompanyAdditionalSettings: Record "GP Company Additional Settings";
+        GPPM00100: Record "GP PM00100";
+        GPPM00101: Record "GP PM00101";
+        GPPM00203: Record "GP PM00203";
+        GPPostingAccounts: Record "GP Posting Accounts";
+        VendorNo: Code[20];
+    begin
+        if not GPCompanyAdditionalSettings.GetGLModuleEnabled() then
+            exit;
+
+        VendorNo := CopyStr(GPVendor.VENDORID, 1, MaxStrLen(VendorNo));
```

Comment:
Can we add the check here to skip the vendor recurring lines in a similar way we did for customers?